src/DSMath.ts

```diff
@@ -136,6 +136,18 @@ export function wdiv(x: BigNumber, y: BigNumber): BigNumber {
   return x.mul(WAD).add(y.div(2)).div(y)
 }
 
+/**
+ * @param {BigNumber} x in WAD
+ * @param {BigNumber} y in WAD
+ * @returns the quotient of x divided by y, in WAD
+ */
+export function safeWdiv(x: BigNumber, y: BigNumber): BigNumber {
+  if (y.lte(constants.Zero)) {
+    return constants.Zero
+  }
+  return x.mul(WAD).add(y.div(2)).div(y)
+}
+
```

Comment:
need safeDiv as well as safeWdiv?

please add test case in DSMath.test.ts to test it doesn't throw when denominator is 0.